deploy/downstream-prep.sh

```diff
@@ -75,39 +75,39 @@ if [ -d deploy/olm-catalog/konveyor-operator/v1.2.4 ]; then
     fi
   done
 
-  # Make 1.2 Downstream CSV Changes
-  for f in deploy/olm-catalog/konveyor-operator/v1.2.4/konveyor-operator.v1.2.4.clusterserviceversion.yaml \
-           deploy/non-olm/v1.2.4/operator.yml
+  # Make 1.3 Downstream CSV Changes
+  for f in deploy/olm-catalog/konveyor-operator/v1.3.0/konveyor-operator.v1.3.0.clusterserviceversion.yaml \
+           deploy/non-olm/v1.3.0/operator.yml
     do
     if [[ "$f" =~ .*clusterserviceversion.* ]]; then
-      sed -i "s,mig-operator-container:.*,openshift-migration-rhel7-operator@sha256:${V1_2_IMG_MAP[operator_sha]},g"                                        ${f}
+      sed -i "s,mig-operator-container:.*,openshift-migration-rhel7-operator@sha256:${V1_3_IMG_MAP[operator_sha]},g"                                        ${f}
     else
-      sed -i "s,mig-operator-container:.*,openshift-migration-rhel7-operator:v1.2,g"                                                                        ${f}
+      sed -i "s,mig-operator-container:.*,openshift-migration-rhel7-operator:v1.3.0,g"                                                                      ${f}
```

Comment:
@rayfordj we will want to set the version of the operator to 1.3.0 in the Dockerfile downstream once the repos/branches are ready. This will hopefully address the 3.x unintentional upgrade issue. fyi @eriknelson